sci-chemistry/ambertools/ambertools-15_p6.ebuild

```diff
@@ -0,0 +1,191 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit fortran-2 python-r1 toolchain-funcs
+
+DESCRIPTION="A suite for carrying out complete molecular mechanics investigations"
+HOMEPAGE="http://ambermd.org/#AmberTools"
+SRC_URI="
+	AmberTools${PV%_p*}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="X"
+
+RESTRICT="fetch"
+
+RDEPEND="${PYTHON_DEPS}
+	virtual/cblas
+	virtual/lapack
+	sci-libs/clapack
+	sci-libs/arpack
+	sci-chemistry/mopac7
+	sci-libs/netcdf
+	sci-libs/netcdf-fortran
+	>=sci-libs/fftw-3.3:3.0
+	sci-chemistry/reduce"
+DEPEND="${RDEPEND}
+	app-shells/tcsh
+	dev-util/byacc
+	dev-libs/libf2c
+	sys-devel/ucpp"
+
+S="${WORKDIR}/amber14"
+
+pkg_nofetch() {
+	einfo "Go to ${HOMEPAGE} and get AmberTools${PV%_p*}.tar.bz2"
+	einfo "Place it into ${DISTDIR}"
+}
+
+pkg_setup() {
+	fortran-2_pkg_setup
+	export AMBERHOME="${S}"
+}
+
+src_prepare() {
```

Comment:
use the PATCHES array variable instead of calling `eapply` manually. Then just put default in `src_prepare()` in the beginning